uppertester/src/main/java/net/gcdc/uppertester/GnTriggerGeoBroadcast.java

```diff
@@ -1,18 +1,6 @@
 package net.gcdc.uppertester;
 
-public class GnTriggerGeoBroadcast {
+public class GnTriggerGeoBroadcast  extends GnTriggerGeoAnycast{
+
     byte messageType = 0x51;
-    byte shape;
```

Comment:
these fields are used by the uppertester/Parser.java, so they should not be deleted.